src/pybind/mgr/cephadm/module.py

```diff
@@ -2275,7 +2289,7 @@ def daemon_action(self, action: str, daemon_name: str, image: Optional[str] = No
 
         if action == 'rotate-key':
             if d.daemon_type not in ['mgr', 'osd', 'mds',
-                                     'rgw', 'crash', 'nfs', 'rbd-mirror', 'iscsi']:
+                                     'rgw', 'crash', 'nfs', 'rbd-mirror', 'iscsi', 'nvmeof']:
```

Comment:
are we sure key rotation actually is going to work for this daemon type?

Reply:
If so, we should probably add it to the test for key rotation as part of this work https://github.com/ceph/ceph/blob/main/qa/suites/orch/cephadm/with-work/tasks/rotate-keys.yaml

Reply:
@adk3798 at this early dev stage I'm not sure this daemon even has key rotation. So I've just removed it from the list. In case it does support the rotation then we can added it to the list.